src/components/Layout.vue

```diff
@@ -1,16 +1,10 @@
 <template>
   <div id="layout" class="d-flex flex-column">
-    <div v-if="networkId && networkId !== 'plasma'" style="background: #FFC107; padding: 0 16px;">
-      <span>{{ $t('components.layout.network') }} {{networkId}}</span>
-    </div>
     <div
-      v-if="foreignNetworkId !== null && (foreignNetworkId != s.ethereum.networkId)"
-      style="background: #FF9800;padding: 16px 16px;"
+      v-if="networkId && networkId !== 'plasma'"
```

Comment:
Why is `plasma` hardcoded here?

Reply:
That's just from the linting. That's the upper notification that will only show on envs other than prod (hence plasma hard coded)

Reply:
This bad network detection still needs to happen in production, because we don't want people signing into BSC while their MetaMask is set to Ethereum, and vice versa.